make OpenBLAS failed with USE_OPENMP=1 and USE_THREAD=0

Hi all, 
I am compiling the OpenBLAS on Ubuntu, however, I got the issue of "OpenBLAS: Cannot set both USE_OPENMP=1 and USE_THREAD=0. The USE_THREAD=0 is only for building single thread version..  Stop." and "recipe for target '/home/ricky/sdpa-c/OpenBLAS/libopenblas.a' failed". I am a beginner of OpenBLAS, could you help me figure this issue? Thank you so much.  Here is the screen output:

================================================

ricky@ricky-virtual-machine:~/sdpa-c$ make OpenBLAS
(cd /home/ricky/sdpa-c; \
rm -rf OpenBLAS; \
git clone git://github.com/xianyi/OpenBLAS; \
mv -f OpenBLAS /home/ricky/sdpa-c/OpenBLAS; \
cd /home/ricky/sdpa-c/OpenBLAS; \
make NO_WARMUP=1 USE_THREAD=1 USE_OPENMP=1 FC=gfortran CC=gcc libs netlib)
Cloning into 'OpenBLAS'...
remote: Counting objects: 32618, done.
remote: Total 32618 (delta 0), reused 0 (delta 0), pack-reused 32618
Receiving objects: 100% (32618/32618), 21.39 MiB | 2.42 MiB/s, done.
Resolving deltas: 100% (26112/26112), done.
Checking connectivity... done.
Checking out files: 100% (9021/9021), done.
mv: cannot move 'OpenBLAS' to a subdirectory of itself, '/home/ricky/sdpa-c/OpenBLAS/OpenBLAS'
make[1]: Entering directory '/home/ricky/sdpa-c/OpenBLAS'
Makefile.system:412: *** OpenBLAS: Cannot set both USE_OPENMP=1 and USE_THREAD=0. The USE_THREAD=0 is only for building single thread version..  Stop.
make[1]: Leaving directory '/home/ricky/sdpa-c/OpenBLAS'
Makefile:83: recipe for target '/home/ricky/sdpa-c/OpenBLAS/libopenblas.a' failed
make: *** [/home/ricky/sdpa-c/OpenBLAS/libopenblas.a] Error 2

It seems that the original problem still exists when I applied the two changes you offered. "OpenBLAS: Cannot set both USE_OPENMP=1 and USE_THREAD=0. The USE_THREAD=0 is only for building single thread version.. Stop." and "recipe for target '/home/ricky/sdpa-c/OpenBLAS/libopenblas.a' failed".

It seems work. But another issue happened. I think the compile of  OpenBLAS has finished. 
===============================================================
g++ -O3 -fPIC -DOPENBLAS -o sdpa-c sdpa_exe.o libsdpa-c.a /home/ricky/sdpa-c/SuiteSparse/CHOLMOD/Lib/libcholmod.a /home/ricky/sdpa-c/SuiteSparse/CCOLAMD/Lib/libccolamd.a /home/ricky/sdpa-c/SuiteSparse/COLAMD/Lib/libcolamd.a /home/ricky/sdpa-c/SuiteSparse/CAMD/Lib/libcamd.a /home/ricky/sdpa-c/SuiteSparse/AMD/Lib/libamd.a /home/ricky/sdpa-c/SuiteSparse/metis-4.0/libmetis.a /home/ricky/sdpa-c/SuiteSparse/SuiteSparse_config/libsuitesparseconfig.a  /home/ricky/sdpa-c/mumps/lib/libdmumps.a /home/ricky/sdpa-c/mumps/lib/libmumps_common.a /home/ricky/sdpa-c/mumps/lib/libpord.a /home/ricky/sdpa-c/mumps/libseq/libmpiseq.a /home/ricky/sdpa-c/OpenBLAS/libopenblas.a -lgomp -lgfortran -lrt
/usr/bin/ld: libsdpa-c.a(sdpa_newton.o): undefined reference to symbol 'pthread_create@@GLIBC_2.2.5'
//lib/x86_64-linux-gnu/libpthread.so.0: error adding symbols: DSO missing from command line
collect2: error: ld returned 1 exit status
Makefile:28: recipe for target 'sdpa-c' failed
make: *** [sdpa-c] Error 1

--- a/Makefile.system
+++ b/Makefile.system
@@ -181,6 +181,7 @@ endif
 
 ifeq ($(NUM_THREADS), 1)
 override USE_THREAD = 0
+override USE_OPENMP = 0
 endif
 
 ifdef USE_THREAD

